### PR TITLE
Add Italian

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

**Comment by matthew-white** [comment]
I've split this PR into two commits. The second just involved pulling the translations and running `destructure.js`. The first sets up i18n for Italian, which was just a few lines. I'll continue updating translations as they're modified and will force-push changes to the second commit. However, this PR should be ready for review.

File: src/components/submission/field-dropdown.vue [hunk 1]
```diff
@@ -302,6 +302,8 @@ export default {
     },
     "disabled": "Cannot select more than 100 columns.",
     "action": {
+      // This text is shown in a dropdown that allows the user to select which
+      // columns to display in a table.
```
Comment (ktuite) [inline src/components/submission/field-dropdown.vue:306]: this doesn't seem related to the Italian translation but i'm assuming you saw other translation things (comments) you wanted to fix up.

**Review by ktuite — APPROVED** [review]
